Assets/MRTK/Core/Utilities/Gltf/Serialization/ConstructGltf.cs

```diff
@@ -458,6 +458,8 @@ private static async Task ConstructNodeAsync(GltfObject gltfObject, GltfNode nod
             var nodeName = string.IsNullOrEmpty(node.name) ? $"glTF Node {nodeId}" : node.name;
             var nodeGameObject = new GameObject(nodeName);
 
+            gltfObject.NodeGameObjectPairs.Add(nodeId, nodeGameObject);
```

Comment:
Do you think there's any benefit to having the dictionary indices be the string names instead of the int ids? I guess all nodes are guaranteed an id while only some may have a name, but the names are a little more human-parseable.

Reply:
Hi, thx for your consideration of my pr! 
As mentioned in the issue #10399, the node names could be the same in the gltf file, that's why I choose the node index as the key. So if we want to make the key as string, we can convert int id to string, but the name could not be an option...

Reply:
Oh, good point! I missed that you had already addressed this in the issue. Sounds good to me!